WordPress/src/main/java/org/wordpress/android/ui/plans/PlanFragment.java

```diff
@@ -8,29 +8,34 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.ImageView;
 import android.widget.LinearLayout;
 import android.widget.TextView;
 
-import com.android.volley.toolbox.NetworkImageView;
-
 import org.apache.commons.text.StringEscapeUtils;
 import org.wordpress.android.R;
 import org.wordpress.android.WordPress;
 import org.wordpress.android.ui.plans.models.Feature;
 import org.wordpress.android.ui.plans.models.Plan;
 import org.wordpress.android.ui.plans.models.PlanFeaturesHighlightSection;
 import org.wordpress.android.util.AppLog;
+import org.wordpress.android.util.image.ImageManager;
+import org.wordpress.android.util.image.ImageType;
 
 import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.HashMap;
 
+import javax.inject.Inject;
+
 public class PlanFragment extends Fragment {
     private static final String SITE_PLAN = "SITE_PLAN";
 
     private ViewGroup mPlanContainerView;
     private Plan mPlanDetails;
 
+    @Inject protected ImageManager mImageManager;
```

Comment:
do we need the `protected` keyword here?

Reply:
I think the visibility modifier should be as strict as possible. Since it's not possible to inject into `private` fields with Dagger 2, `protected` is the second most strict modifier. Do you prefer to keep it package private?

Reply:
`protected` sounds good in that case -  it was just that it looked odd as my first thought is  that there was an intention to expose this property to derived classes (hence my question), rather than trying to keep it from being used elsewhere (as `private` would be). It's ok as is I think!